lib/google/ads/google_ads/lookup_service.rb

```diff
@@ -0,0 +1,154 @@
+require 'google/ads/google_ads/patch_lro_headers'
+
+module Google
+  module Ads
+    module GoogleAds
+      class LookupService
```

Comment:
Can you name this "ServiceLookup" instead. "LookupService" reads to me like "a service for looking things up", rather than what this actually is, which is a util that looks up a service.